db/models/curator-model.js

```diff
@@ -72,6 +72,11 @@ class Curators extends Model {
       foreignKey: "address",
       targetKey: "address"
     });
+
+    this.hasMany(models.user, {
```

Comment:
I think this relation should be `belongsTo`, because a record on curator table is related to only a user. But is there a specific reason for this relation to be `hasMany` ?

Reply:
this was a leftover from a talk in the daily ^^'